dive/image/layer.go

```diff
@@ -39,5 +41,5 @@ func (l *Layer) String() string {
 	}
 	return fmt.Sprintf(LayerFormat,
 		humanize.Bytes(l.Size),
-		l.Command)
+		strings.Split(l.Command, "\n")[0])
```

Comment:
Consider using the solution from https://github.com/wagoodman/dive/pull/443 instead - replace newlines with the ↵ character but still show the whole command, in case the first line isn't very informative.

(Also, FYI, `strings.Split(...)[0]` is equivalent to `strings.Cut`)